Mac.sln

```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
why do we need a new sln, can this not be part of the existing one?

Reply:
I think this is fine. It's the easiest way to exclude projects taht don't build on Mac.